optimum/exporters/onnx/model_configs.py

```diff
@@ -769,9 +769,27 @@ class GroupViTOnnxConfig(CLIPOnnxConfig):
     pass
 
 
-# TODO: not supported now because of aten:broadcast_to, can be most likely patched.
-# class OwlViTOnnxConfig(CLIPOnnxConfig):
-#     pass
+class OwlViTOnnxConfig(CLIPOnnxConfig):
+
+    # Sets the absolute tolerance to when validating the exported ONNX model against the
+    # reference model.
+    ATOL_FOR_VALIDATION = 1e-4
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        return {**super().inputs}
+    
+    @property
+    def outputs(self) -> Dict[str, Dict[int, str]]:
+        # TODO what is feature-extraction return value? 
```

Comment:
For feature extraction, I see [here](https://github.com/huggingface/transformers/blob/3658488ff77ff8d45101293e749263acf437f4d5/src/transformers/models/owlvit/modeling_owlvit.py#L1231) in Transformers that the model returns `logits_per_image`, `logits_per_text`, `text_embeds` and `image_embeds`.
You can easily define different outputs for different tasks using the `task` attribute (see an example with BART [here](https://github.com/huggingface/optimum/blob/a6951c17c3450e1dea99617aa842334f4e904392/optimum/exporters/onnx/model_configs.py#L458)).

Reply:
I made it as an if and elif. Should I make it as an if and else?
Which one should be the else?

Reply:
I think it's fine like this. If another task is given, an error should be raised before even arriving here I believe.